data/auxiliary/gather/ldap_query/ldap_queries_default.yaml

```diff
@@ -10,19 +10,42 @@ queries:
       - cACertificateDN
       - dNSHostname
       - certificateTemplates
-  - action: ENUM_ADCS_CERT_TEMPLATES
```

Comment:
Can you put this query to list all certificate templates back in please?

Reply:
Sure I'll do that and remove the EC1 one since that will need to go into another PR

Reply:
Restored with latest updates, specifically https://github.com/rapid7/metasploit-framework/pull/17071/commits/abd8ad1d7d607a0c206f43de5bd99cf4b8b42e0c